### PR TITLE
Mforets/update arch

### LINKED ISSUE #155: Update nonlinear models
Cf. https://github.com/JuliaReach/Reachability.jl/pull/657

### PR DESCRIPTION
**Pull request by mforets**
Closes #155.

### CODE REVIEW

File: models/ARCH/AFF/SpaceStation/iss_benchmark.jl [hunk 1]
```diff
@@ -39,10 +54,10 @@ projection_matrix = C[:, 136:270]
 𝑂_ISS01[:projection_matrix] = projection_matrix
 opC = BFFPSV18(merge(𝑂_dense_ISS01, Options(:block_options_iter => nothing)))
 res = solve(iss_TV, 𝑂_ISS01, op=opC)
-res = ReachSolution(
-    [ReachSet(Interval(rs.t_start, rs.t_end) × rs.X, rs.t_start, rs.t_end)
-        for rs in res.Xk],
-    Options(:plot_vars => [0, 1]))
+
+# FIXME: use Reachability's project function
+res = ReachSolution(add_time(res.Xk), Options(:plot_vars => [0, 1]))
```
Comment (schillic) [inline models/ARCH/AFF/SpaceStation/iss_benchmark.jl]: I do not understand why you cannot do "use Reachability's project function".

Comment (mforets) [reply models/ARCH/AFF/SpaceStation/iss_benchmark.jl]: There are several errors with that function, see https://github.com/JuliaReach/Reachability.jl/issues/707

**Comment by mforets** [comment]
> I am fine with most changes, but the ISS change seems like a step backward.

ISS is one of the models fixed by this PR.

File: models/ARCH/AFF/SpaceStation/iss_benchmark.jl (hunk 1 above)
Comment (schillic) [reply models/ARCH/AFF/SpaceStation/iss_benchmark.jl]: Okay, but then I would prefer to fix those errors instead of adding workarounds in the model files.

**Pull request merged**